charts/templates/statefulset.yaml

```diff
@@ -29,24 +29,28 @@ spec:
           - sh
           - '-c'
           - |
-            echo '### Begin configuring group ###'
-
-            HOST_RESP_CODE=`curl --anyauth -m 20 -s -o /dev/null -w "%{http_code}" -X GET http://${MARKLOGIC_BOOTSTRAP_HOST}:8002/manage/v2/hosts --anyauth --user ${MARKLOGIC_ADMIN_USERNAME}:${MARKLOGIC_ADMIN_PASSWORD}`
-            if [[ ${HOST_RESP_CODE} -eq 200 ]]; then
-              GROUP_CFG_TEMPLATE='{"group-name":"%s"}'
-              GROUP_CFG=$(printf "$GROUP_CFG_TEMPLATE" "$MARKLOGIC_GROUP")
-              GROUP_RESP_CODE=`curl --anyauth -m 20 -s -o /dev/null -w "%{http_code}" -X GET http://${MARKLOGIC_BOOTSTRAP_HOST}:8002/manage/v2/groups/${MARKLOGIC_GROUP} --anyauth --user ${MARKLOGIC_ADMIN_USERNAME}:${MARKLOGIC_ADMIN_PASSWORD}`
-
-              if [[ ${GROUP_RESP_CODE} -eq 200 ]]; then
-                echo "Group $MARKLOGIC_GROUP is already configured on the MarkLogic cluster." 
-              else
-                echo "Group $MARKLOGIC_GROUP does not exist, creating $MARKLOGIC_GROUP on the MarkLogic cluster"
-                curl --anyauth --user ${MARKLOGIC_ADMIN_USERNAME}:${MARKLOGIC_ADMIN_PASSWORD} -m 20 -s -X POST -d "${GROUP_CFG}" -H "Content-type: application/json" http://${MARKLOGIC_BOOTSTRAP_HOST}:8002/manage/v2/groups
-              fi
+            echo '### Begin init container execution ###'
```

Comment:
Please clean up the log messages to match other scripts. Add the "log" function:
```
                    log () {
                      local TIMESTAMP=$(date +"%Y-%m-%d %T.%3N")
                      echo "${TIMESTAMP}  $@"
                    }
```
Use log messages with Info or Error and make sure it is clear to the user why the messages are being logged.